drivers/wdm/audio/backpln/portcls/port_topology.cpp

```diff
@@ -197,7 +186,7 @@ CPortTopology::GetDeviceProperty(
 
     if (!m_bInitialized)
     {
-        DPRINT("IPortTopology_fnNewRegistryKey called w/o initialized\n");
+        DPRINT("IPortTopology_fnNewRegistryKey called w/o initiazed\n");
```

Comment:
Skip that change. It is wrong.

Reply:
I don't see that misspelling in the code.